packages/components-react/src/examples/ModalExampleChangeContent.tsx

```diff
@@ -0,0 +1,47 @@
+import { useCallback, useState } from 'react';
+import { PButton, PModal, PText, PButtonGroup, PCheckboxWrapper } from '@porsche-design-system/components-react';
+
+export const ModalExampleChangeContentPage = (): JSX.Element => {
```

Comment:
This component should go into pages since examples imply that they are used in the storefront.